### PR TITLE
feat: text area component

### PR DESCRIPTION
**Pull request by jessywork**
This update addresses the need for a customizable and reusable set of a textarea component in the project.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by joaoprcosta** [comment]
Remove CHANGELOG. Is is auto generated after published

File: packages/components/textarea/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export { default as Textarea } from './Textarea';
```
Comment (joaoprcosta) [inline packages/components/textarea/src/index.ts:1]:
```suggestion
export { default } from './Textarea';
```

File: packages/components/textarea/src/Textarea.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      <TextareaHolderStyled
+        styled={mergedThemeValues as Required<TextareaStyledProps>}
+      >
+        <TextareaStyled
+          maxLength={maxLength}
+          placeholder={placeholder}
+          styled={mergedThemeValues as Required<TextareaStyledProps>}
+          {...props}
+        >
+          {props.children}
```
Comment (joaoprcosta) [inline packages/components/textarea/src/Textarea.tsx:54]: I think children will not be needed in a textarea.

File: packages/components/textarea/src/textarea.styles.ts [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    textarea {
+      color: ${({ styled }) => styled.colorActive};
+    }
+  }
+`;
+
+export const TextareaStyled = styled.textarea<{
+  styled: Required<TextareaStyledProps>;
+}>`
+  font-size: 0.875rem;
```
Comment (joaoprcosta) [inline packages/components/textarea/src/textarea.styles.ts:57]: Should font-size be fed by theme?

**Review by JuniorDiasOliveira — APPROVED** [review]

File: packages/components/textarea/src/Textarea.tsx [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+          styled={mergedThemeValues as Required<TextareaStyledProps>}
+        >
+          {label}
+        </TextareaLabelStyled>
+      )}
+
+      <TextareaHolderStyled
+        styled={mergedThemeValues as Required<TextareaStyledProps>}
+      >
+        <TextareaStyled
```
Comment (joaoprcosta) [inline packages/components/textarea/src/Textarea.tsx:49]: <TextareaStyled
          maxLength={maxLength}
          placeholder={placeholder}
          styled={mergedThemeValues as Required<TextareaStyledProps>}
          {...props}
        />

**Review by joaoprcosta — APPROVED** [review]

**Pull request merged**